CommentingPractice/model/food.js

```diff
@@ -0,0 +1,128 @@
+/**
```

Comment:
Please take a look at this page about inheritance: https://javascript.info/class-inheritance